components/molecule/notification/src/index.scss

```diff
@@ -0,0 +1,105 @@
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/index';
```

Comment:
you should be able to create this component without using the old vars, please make it work without this line